pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTaskRestletResource.java

```diff
@@ -553,9 +553,9 @@ public List<JobKey> getCronSchedulerJobKeys()
 
   @GET
   @Path("/tasks/scheduler/jobDetails")
-  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Cluster.GET_SCHEDULER_INFO)
+  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Table.GET_SCHEDULER_JOB_DETAILS)
```

Comment:
given table name is required, should targetType be TABLE here?

Reply:
Ah good catch! Made the change.